e2e/tests/visual-a11y/planning.visual.spec.js

```diff
@@ -32,7 +32,7 @@ const examplePlanSmall = require('../../test-data/examplePlans/ExamplePlan_Small
 
 const snapshotScope = '.l-shell__pane-main .l-pane__contents';
 
-test.describe('Visual - Planning', () => {
+test.describe('Visual - Planning @a11y', () => {
```

Comment:
Do we need the `Visual -` prefix anymore? maybe we can just use the `@a11y` tag now

Reply:
i'll keep it